proxy.js

```diff
@@ -14,6 +13,9 @@ function reset() {
 }
 
 module.exports = {
+  setSaveDir(saveDir) {
+    pacScriptPath = path.join(saveDir, "proxy-pac.js");
+  },
```

Comment:
@fjvallarino no need to change in this PR, but in the future, I think we should try to move away from storing state in components and instead provide arguments.